l3build.lua

```diff
@@ -24,50 +24,207 @@ for those people who are interested.
 
 --]]
 
+print("DEVELOPMENT: REFACTOR BRANCH")
+
 -- Version information
 release_date = "2020-06-04"
 
--- File operations are aided by the LuaFileSystem module
-local lfs = require("lfs")
-
 -- Local access to functions
 
-local assert           = assert
-local ipairs           = ipairs
-local insert           = table.insert
-local lookup           = kpse.lookup
-local match            = string.match
-local gsub             = string.gsub
-local next             = next
-local print            = print
-local select           = select
-local tonumber         = tonumber
-local exit             = os.exit
-
--- l3build setup and functions
+local assert  = assert
+local ipairs  = ipairs
+local append  = table.insert
+local match   = string.match
```

Comment:
It is weird to use a command named `insert` to append.
```
append(foo, bar)
```
reads better than
```
insert(foo, bar)
```
at least because you have to remember that insertion is at the en by default.